services/121-service/src/payments/payments.service.spec.ts

```diff
@@ -17,13 +25,13 @@ function createMockTransaction(id: number, amount: number, status: string) {
     registrationProgramId: id + 100,
     referenceId: `REF-${id}`,
     errorMessage: null,
-    programFinancialServiceProviderConfigurationLabel: { en: 'Some FSP Label' },
+    programFinancialServiceProviderConfigurationName: 'Some FSP',
```

Comment:
```suggestion
    programFinancialServiceProviderConfigurationName: 'someFsp',
```

Reply:
It's a bit better because than it's cleare that this is a name/key and not a label